### PR TITLE
test: add new test case to clients/imagebuilder/client.go

### PR DESCRIPTION
**Pull request by adeliaferreira**
Current for this file, our coverage is equal 52.32% , Adding a new test case to increase this file coverage.

Signed-off-by: Adelia Ferreira <tay.figueira@gmail.com>

FIXES: 3008

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Tests update
- [ ] Refactor

### CODE REVIEW

File: pkg/clients/imagebuilder/client_test.go [hunk 1]
```diff
@@ -143,6 +143,44 @@ var _ = Describe("Image Builder Client Test", func() {
 		Expect(img.Commit.ComposeJobID).To(Equal("compose-job-id-returned-from-image-builder"))
 		Expect(img.Commit.ExternalURL).To(BeFalse())
 	})
+	It("test get thirds party repo without orgId", func() {
```
Comment (ldjebran) [inline pkg/clients/imagebuilder/client_test.go:146]: what is the goal of this test?

File: pkg/clients/imagebuilder/client_test.go [hunk 2]
```diff
@@ -143,6 +143,44 @@ var _ = Describe("Image Builder Client Test", func() {
...
+		ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```
Comment (ldjebran) [inline pkg/clients/imagebuilder/client_test.go:147]: GetImageThirdPartyRepos func does not call any server , why we need to create this http server?

Comment (adeliaferreira) [reply pkg/clients/imagebuilder/client_test.go]: You're right, removed it from the test

File: pkg/clients/imagebuilder/client_test.go (hunk 1 above)
Comment (adeliaferreira) [reply pkg/clients/imagebuilder/client_test.go]: Updated test description to a better description. The goal for this test is validate the error is occurred when try to retrieve thirdpartyrepos without orgId associated on the image

File: pkg/clients/imagebuilder/client_test.go [hunk 3]
```diff
@@ -143,6 +143,36 @@ var _ = Describe("Image Builder Client Test", func() {
...
+				},
+				{
+					Name: "repo test2",
+					URL:  "https://repo2.com",
+				},
+			},
+		}
+		result, err := client.GetImageThirdPartyRepos(img)
+		Expect(result).To(BeNil())
+		Expect(err).To(HaveOccurred())
```
Comment (ldjebran) [inline pkg/clients/imagebuilder/client_test.go:174]:
```suggestion
		Expect(err).To(HaveOccurred())
		Expect(err.Error()).To(Equal("error retrieving orgID  information, image orgID undefined"))
```

**Review by ldjebran — APPROVED** [review]
lgtm

**Review by AaronH88 — APPROVED** [review]
/lgtm

**Pull request merged**